From e68fa8d000ad9d01719a6a628b42878cb851c707 Mon Sep 17 00:00:00 2001
From: Miklos Csanady <miklos.csanady@cloudera.com>
Date: Mon, 27 Mar 2017 10:08:28 +0200
Subject: [PATCH 1049/1164] CLOUDERA-BUILD: CDH-51603: S3 Tests: Investigate
 if hive.blobstore.optimizations.enabled needs to
 be set to false

Change-Id: I45c88833618529fdc5f6cc02ee8e829b8529079c
---
 .../resources/hdfs-blobstore/blobstore-conf.xml    |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/itests/hive-blobstore/src/test/resources/hdfs-blobstore/blobstore-conf.xml b/itests/hive-blobstore/src/test/resources/hdfs-blobstore/blobstore-conf.xml
index 288b81e..06b5b1b 100644
--- a/itests/hive-blobstore/src/test/resources/hdfs-blobstore/blobstore-conf.xml
+++ b/itests/hive-blobstore/src/test/resources/hdfs-blobstore/blobstore-conf.xml
@@ -12,10 +12,6 @@
     <value>s3a://cloudera-dev-hive-on-s3/core-blobstore-path/</value>
   </property>
   <property>
-    <name>hive.blobstore.optimizations.enabled</name>
-    <value>false</value>
-  </property>
-  <property>
     <name>additional.plan.masks</name>
     <value>s3.*</value>
   </property>
-- 
1.7.9.5

